ccpage146.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Write a function called city_country() that takes in the name of a city and its country.
+# The function should return a string formatted like this: "Santiago, Chile"
+# Call your function with at least three city-country pairs, and print the value that’s returned.
+Option 1
+def city_country (city, country):
+    """List a city and its country"""
+    print(f"{city.title()}, {country.title()}")
+
+city_country('dublin', 'ireland')
+city_country('boston', 'united states')
+city_country('rome', 'italy')
+
+Option 2: 
+def city_country(city, country):
+    """Locate city and its country"""
+    location = city + ', ' + country
+    return location.title()
+
+city_country_location = city_country('rome', 'italy')
+print(city_country_location)
+
+city_country_location = city_country('boston', 'united states')
+print(city_country_location)
+
+city_country_location = city_country('dublin', 'ireland')
+print(city_country_location)
```

Comment:
Nice practices on these saving the return value as a variable and doing something (ie, printing) with it.